src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -101,13 +104,18 @@ WHERE dar.submission_date > now() - interval '1 year'
               SELECT dar.id, dar.reference_id, dar.collection_id, dar.parent_id,
                 dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date,
                 (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data,
-                dd.dataset_id
+                dd.dataset_id, collection.dar_code
               FROM data_access_request dar
+              LEFT JOIN dar_collection collection on collection.collection_id = dar.collection_id
               LEFT JOIN dar_dataset dd ON dd.reference_id = dar.reference_id
-              WHERE dar.submission_date BETWEEN SYMMETRIC :begin AND :end
+              LEFT OUTER JOIN email_entity email ON email.entity_reference_id = dar.reference_id AND email.email_type = :emailType
+              WHERE dar.submission_date >= :notBefore
+              AND (dar.submission_date < now() - :interval ::interval)
+              AND (email.email_type IS NULL)
```

Comment:
I'm confused by this restriction on null email type. How does this work if we're passing in an email type? The left outer join on line 111 along with this condition is counter-intuitive.

Reply:
I use the email type in the left outer join to enrich the result set with the messages we've sent matching the email type, then this code filters those rows because we've already sent that type of message.  The net effect is that we get a list of DARs we haven't sent the emailType notice to yet.  :-)

Reply:
> The net effect is that we get a list of DARs we haven't sent the emailType notice to yet. :-)

Thank you, that makes sense 👍🏽